code/jobs/RunBuildTaskJob.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * A job used to delete a data object. Typically used for deletes that need to happen on
+ * a schedule, or where the delete may have some onflow affect that takes a while to
+ * finish the deletion.
```

Comment:
Is this doc block correct? Looks like the job is for running rather than deleting